truss/cli/cli.py

```diff
@@ -304,7 +306,13 @@ def _extract_request_data(data: Optional[str], file: Optional[Path]):
     "--model-version",
     type=str,
     required=False,
-    help="ID of model version to invoke",
+    help="[DEPRECATED] Use --model-deployment instead, this will be removed in future release. ID of model version",
+)
+@click.option(
+    "--model-deployment",
+    type=str,
+    required=False,
+    help="ID of model deployment to invoke",
```

Comment:
We standardized on "call" instead of "invoke" in the product, would be good to match here.

Reply:
cool will change